ocs_ci/ocs/scale_lib.py

```diff
@@ -82,150 +77,151 @@ def create_and_set_namespace(self):
-            dc_deployment=self.dc_deployment,
-            raw_block_pv=True,
-            node_selector=self.node_selector,
+
+        # Create kube_job for pod creation
+        lcl[f"pod_kube_{obj_name}"] = ObjectConfFile(
+            name=f"pod_kube_{obj_name}",
+            obj_dict_list=pod_data_list,
+            project=self.namespace,
+            tmp_path=tmp_path,
         )
-        temp_pod_objs = list()
-        temp_pod_objs.extend(cephfs_pods + rbd_rwo_pods)
-
-        # Appending all the pod_obj to list
-        pod_objs.extend(temp_pod_objs + rbd_rwx_pods)
-
-        # Start IO
-        if start_io:
-            threads = list()
-            for pod_obj in temp_pod_objs:
-                process = threading.Thread(
-                    target=pod_obj.run_io,
-                    kwargs={
-                        "storage_type": "fs",
-                        "size": fio_size,
-                        "runtime": io_runtime,
-                        "rate": fio_rate,
-                    },
-                )
-                process.start()
-                threads.append(process)
-                time.sleep(30)
-            for pod_obj in rbd_rwx_pods:
-                process = threading.Thread(
-                    target=pod_obj.run_io,
-                    kwargs={
-                        "storage_type": "block",
-                        "size": fio_size,
-                        "runtime": io_runtime,
-                        "rate": fio_rate,
-                    },
-                )
-                process.start()
-                threads.append(process)
-                time.sleep(30)
-            for process in threads:
-                process.join()
+        lcl[f"pod_kube_{obj_name}"].create(namespace=self.namespace)
 
-        return pod_objs, pvc_objs
+        # Check all the POD reached Running state
+        pod_running_list = check_all_pod_reached_running_state_in_kube_job(
+            kube_job_obj=lcl[f"pod_kube_{obj_name}"],
+            namespace=self.namespace,
+            no_of_pod=len(pod_data_list),
+            timeout=90,
+        )
 
-    def create_scale_pods(
-        self,
-        scale_count=1500,
-        pods_per_iter=5,
-        io_runtime=None,
-        pvc_size=None,
-        start_io=None,
-    ):
+        # Update list with all the kube_job object created, list will be
+        # used in cleanup
+        self.kube_job_pvc_list.append(lcl[f"rbd_pvc_kube_{obj_name}"])
+        self.kube_job_pvc_list.append(lcl[f"cephfs_pvc_kube_{obj_name}"])
+        self.kube_job_pod_list.append(lcl[f"pod_kube_{obj_name}"])
+
+        return rbd_pvc_name, fs_pvc_name, pod_running_list
+
+    def create_scale_pods(self, scale_count=1500, pvc_per_pod_count=20):
         """
         Main Function with scale pod creation flow and checks to add nodes.
         For other platforms will not be considering the instance_type param
+        Function does scale pods in counts in 750 and more lesser than 750
```

Comment:
`scale_count` can be taken for pods, but they are pvcs. should we update this arg to `pvc_count`?
Also, the docstring `Function does scale pods` is misleading.

Reply:
yes, will update the doc string